### ISSUE TITLE
pre-training model

### ISSUE THREAD

**Issue by lcx0329** [j-min/CLIP-Caption-Reward#12]
Excuse me, is the downloaded pre-training model already trained? Why is the test result shown in the figure below?
![image](https://github.com/j-min/CLIP-Caption-Reward/assets/69033379/d664682f-d58d-4ca5-bcc9-4d1ada575b62)

**Comment by j-min** [comment]
I don't have enough information to guide this just from the screenshot, but maybe you can first check the generated outputs? Try [colab demo](https://colab.sandbox.google.com/github/j-min/CLIP-Caption-Reward/blob/main/Inference_example.ipynb) and see if your generation results are similar to the generation results at colab. The colab is based on the pretrained checkpoint.

**Comment by lcx0329** [comment]
Due to a mismatch between the PL library and Pytorch, I upgraded the PL library, which resulted in some code changes that needed to be made. After the changes were made, the results are shown in the figure. Here are the modifications I made, and I would like to ask if I have made any mistakes
![image](https://github.com/j-min/CLIP-Caption-Reward/assets/69033379/06de2a84-1f00-440f-8ffc-09dd784b9a2e)
The above is the modification I made. Could you please check if it was due to the above modifications? Thank you

**Comment by j-min** [comment]
I don't see any obvious error from the screenshot, but I don't have information about all the changes you might have made to give useful comments on your case.
I suggest starting from the current pytorch / lightning version in this repo so that you have a clean starting point, and gradually changing one thing at a time to spot what makes the performance drop.

**Comment by lcx0329** [comment]
![image](https://github.com/j-min/CLIP-Caption-Reward/assets/69033379/82ab3ef8-b13e-4463-a841-a706b605bd26)
The above evaluation results, I don't know why every sentence is incomplete
